doc/buildAndProgram.md

```diff
@@ -14,7 +14,7 @@ To build this project, you'll need:
  - A reasonably recent version of CMake (I use 3.16.5)
  - lv_font_conv, to generate the font .c files
    - see [lv_font_conv](https://github.com/lvgl/lv_font_conv#install-the-script)
-   - install npm (commonly done via the package manager)
+   - install npm (commonly done via the package manager, might need to upgrade node if you get `TypeError: this.src.glyphs.map(...).flat is not a function`)
```

Comment:
when do you get the error message? is it during `npm install` or when running `lv_font_conv` executable?

for the first case the warning should be moved to the `npm install` command

if the warning is issued during `lv_font_conv` execution, then I'd add the warning as a new bulletpoint after the `npm install` command

Reply:
I think it should just say: install npm and make sure node is version 12 or newer. Having the note about the version requirement later doesn't seem right to me.

Reply:
changed to "install npm (commonly done via the package manager. ensure the version is at least 12)" and that's it.
It does look much neater.

Reply:
Node needs to be mentioned however, since node and npm have different versions.

Reply:
I wouldn't add it. For me "package manager" is `apt`, `dnf` or `pacman`